README.md

```diff
@@ -1 +1,46 @@
-# Future Debian/Ubuntu package for The Lounge
+# Debian/Ubuntu package for The Lounge
+This repository holds out the build scripts that generates our .deb precompiled packages and also tracks Debian-specific issues in relation to the packaging.
```

Comment:
Let's get started with the nitpicky comments :-)

Can you add an empty line between the headers (`#`, `##`, ...) and the text?
I know some Markdown parsers don't handle this well (like npm's one is really bad if your markup isn't perfect lol) and it could lead to weird things to fix if we start publishing this repo somewhere.